blocks/procedures.js

```diff
@@ -933,10 +933,10 @@ Blockly.Blocks['procedures_callnoreturn'] = {
         }
         Blockly.Events.setGroup(event.group);
         if (event.newValue) {
-          this.previousDisabledState_ = this.disabled;
-          this.setDisabled(true);
+          this.previousEnabledState_ = this.isEnabled();
```

Comment:
There's a couple other references to previousDisabledState_ in this file as well.